openshift/templates/iiw-book-agent/iiw-book-agent-deploy.dev.param

```diff
@@ -0,0 +1,29 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: .
+# Template File: ../openshift/templates/iiw-book-agent/iiw-book-agent-deploy.yaml
+#=========================================================
+# NAME=iiw-book-agent
+# APP_GROUP=iiw-book
+NAMESPACE_NAME=a99fd4
+# ROLE=agent
+APPLICATION_DOMAIN=iiw-book-agent-dev.apps.silver.devops.gov.bc.ca
+APPLICATION_ADMIN_DOMAIN=iiw-book-agent-admin-dev.apps.silver.devops.gov.bc.ca
+IMAGE_NAMESPACE=a99fd4-tools
+TAG_NAME=dev
+# INDY_ADMIN_PORT=4000
+# INDY_AGENT_PORT=11000
+# ADMIN_API_KEY=
+# WALLET_KEY=
+# WALLET_SEED=[a-zA-Z0-9]{32}
+AGENT_URL=https://iiw-book-agent-dev.apps.silver.devops.gov.bc.ca
+# WEBHOOK_URL=http://iiw-book-service:8080/webhooks
+GENESIS_FILE_URL=https://raw.githubusercontent.com/sovrin-foundation/sovrin/stable/sovrin/pool_transactions_sandbox_genesis
+# DISPLAY_LABEL=ConfBook
+# MOUNT_PATH=/home/indy/.indy_client/wallet
+# PERSISTENT_VOLUME_SIZE=5Gi
+# PERSISTENT_VOLUME_CLASS=netapp-file-standard
+# CPU_REQUEST=10m
+# CPU_LIMIT=250m
+# MEMORY_REQUEST=10Mi
+# MEMORY_LIMIT=1Gi
```

Comment:
Resource limits in non-local parameter files should not be commented out: only the `.local` files have them set to 0 by default, "regular" param files are best when explicitly spcifying the limits (pay attention to other param files where these are set to 0 as they will need updating).